src/containerapp/azext_containerapp/custom.py

```diff
@@ -445,7 +447,8 @@ def create_containerapp(cmd,
                         transport="auto",
                         ingress=None,
                         allow_insecure=False,
-                        revisions_mode="single",
+                        revisions_mode=None,
```

Comment:
Why change the default value of `revision-mode`? It is a breaking change.

![image](https://github.com/user-attachments/assets/6fdf3791-bfd8-426e-a93b-d24c23f841ec)

Reply:
The default is not actually changing, 'single' is still applied as the default in the CP. See https://github.com/Azure/azure-cli-extensions/pull/8324#discussion_r1859074278